src/pages/CheckoutSecond.page.js

```diff
@@ -0,0 +1,60 @@
+const { ShopingCartPage } = require('./ShopingCart.page');
+
+export class CheckoutSecondPage extends ShopingCartPage {
+    url = '/checkout-step-two.html';
+
+    get headerTitle() { return this.page.locator('.title'); }
+
+    get finishButton() { return this.page.locator('[data-test="finish"]'); }
+
+    get checkoutItems() { return this.page.locator('.cart_item'); }
+
+    get itemsTotal() { return this.page.locator('.summary_subtotal_label'); }
+
+    get itemsTax() { return this.page.locator('.summary_tax_label'); }
+
+    get orderTotalPrice() { return this.page.locator('.summary_total_label'); }
+
+    async goToFinishCheckoutFlow() {
+        await this.finishButton.click();
```

Comment:
I wouldn't wrap one liner with other one liner, since it adds not a lot of sence. But here we have descriptive name, so IDK :)